### ISSUE TITLE
I am using BitBucket but I am getting the "GitHub API’s rate limit" (403) error

### ISSUE THREAD

**Issue by jordymeow** [afragen/github-updater#761]
Hi,

This plugin seems awesome, I a looking forward to being able to use it :) However, I am using it with BitBucket and it seems like it might be trickier...

I have added a plugin through GitHub Updater, and it worked. I pushed a new version to test the plugin but I am getting this error:

```
GitHub Updater Error Code: 403	
GitHub API’s rate limit will reset in 41 minutes.
It looks like you are running into GitHub API rate limits. Be sure and configure a Personal Access Token to avoid this issue.
```

I don't think I should get an error related to GitHub API since I am not using it, is this a bug? How can I go around it?

Thanks a lot.

**Comment by afragen** [comment]
But you are using GitHub’s API. GitHub Updater is on GitHub.

Under normal circumstances you shouldn’t see this if you don’t have any other projects on GitHub. You must be using the Refresh Cache a lot. 

If you have an account on GitHub just create a personal access token and add it to the GitHub Settings. You’ll never see that notice again.

**Comment by afragen** [comment]
FYI this will have no effect on querying Bitbucket’s API. Do you have any issues with the plugin working?

**Comment by jordymeow** [comment]
Indeed, this removes the message, I didn't realize that the plugin itself was looking for its own updates :)

I could use it for one plugin I have on BitBucket. I had to use the filter to avoid the updates to go through the WordPress Repository (not sure why I had to do that; I added the plugin through GitHub Updater, so it should be managed by it automatically I think). However, I have added another plugin that I have on BitBucket and something odd happened. The plugin was installed, but all the information about the two plugins disappeared from GitHub Updater, even the little BitBucket tab that was there previously :(
